api/v1_playlists_trending.go

```diff
@@ -112,6 +112,8 @@ func (app *ApiServer) v1PlaylistsTrending(c *fiber.Ctx) error {
 			MyID: myId,
 		},
 		OmitTracks: params.OmitTracks,
+		// Limit these to 5 items to prevent slow load times
+		TrackLimit: 5,
```

Comment:
we have UI that will show +X other tracks right?

Reply:
Yes, it's based on the `track_count` field, which you just helped me realize was being computed based on the wrong data. So...good catch!